libs/blocks/marquee/marquee.js

```diff
@@ -80,6 +80,16 @@ export async function loadMnemonicList(foreground) {
   }
 }
 
+export async function loadCountDownTimer(el) {
+  const { base } = getConfig();
+  const stylePromise = new Promise((resolve) => {
+    loadStyle(`${base}/blocks/countdown-timer/countdown-timer.css`, resolve);
+  });
+  const loadModule = import('../countdown-timer/countdown-timer.js')
+    .then(() => { const countDownTimer = el.querySelector('.countdown-timer'); loadBlock(countDownTimer); });
+  await Promise.all([stylePromise, loadModule]);
+}
+
```

Comment:
We should embed this within the module, that way we just need to load the module in a non blocking way and pass the element `el.querySelector('.countdown-timer')` into it.

Reply:
Updated